plugins/modules/zos_fetch.py

```diff
@@ -170,66 +167,66 @@
 
 RETURN = r"""
 file:
-    description: The source file path on remote machine
+    description: The source file path or data set on the remote machine.
     returned: success
     type: str
     sample: SOME.DATA.SET
 dest:
-    description: The destination file path on controlling machine
+    description: The destination file path on the controlling machine.
     returned: success
     type: str
     sample: /tmp/SOME.DATA.SET
 is_binary:
-    description: Indicates which transfer mode was used to fetch the file
+    description: Indicates the transfer mode that was used to fetch.
     returned: success
     type: bool
     sample: True
 checksum:
-    description: The SHA256 checksum of the fetched file
+    description: The SHA256 checksum of the fetched file.
     returned: success and src is a non-partitioned data set
     type: str
     sample: 8d320d5f68b048fc97559d771ede68b37a71e8374d1d678d96dcfa2b2da7a64e
 data_set_type:
-    description: Indicates the fetched file's data set type
+    description: Indicates the fetched data set type.
     returned: success
     type: str
     sample: PDSE
 note:
-    description: Notice of module failure when C(fail_on_missing) is false
+    description: Notice of module failure when C(fail_on_missing) is false.
     returned: failure and fail_on_missing=false
     type: str
     sample: The data set USER.PROCLIB does not exist. No data was fetched.
 changed:
-    description: Indicates if any changes were made during the module operation
+    description: Indicates if any changes were made during the module operation.
```

Comment:
@asifmahmud 
I think we can remove changed and only document change when we differ outside the norm, ie something like command execution where we really don't know if commands are successful but only that they are successfully submitted there for changed would always be true, that is a good example. 

In this modules description it seems fetching would never make a change and conforms to the same theme of how Ansible documents it :https://docs.ansible.com/ansible/latest/reference_appendices/common_return_values.html#id11

Reply:
@ddimatos - Yeah, I think that would be better. We don't actually have to document this parameter, since it gets reported to the user anyway